src/ontology/mondo-edit.obo

```diff
@@ -269696,38 +269696,29 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
 
 [Term]
 id: MONDO:0011897
-name: leukoencephalopathy-ataxia-hypodontia-hypomyelination syndrome
-def: "A syndrome is characterized by ataxia, delayed dentition, hypomyelination and cerebral atrophy. So far, eight cases have been described." [https://orcid.org/0000-0001-5208-3432, Orphanet:137639]
```

Comment:
This definition should be updated instead of deleted.

Reply:
I added a def